src/status_im/navigation/effects.cljs

```diff
@@ -117,8 +134,10 @@
 
 (defn open-modal
   [component]
-  (let [{:keys [options]} (get views/screens component)
-        sheet?            (:sheet? options)]
+  (let [{:keys [options name]} (get views/screens component)
+        sheet?                 (:sheet? options)
+        screen-theme           (:theme options)]
+    (set-status-bar-color screen-theme name)
```

Comment:
hi, why we still need to add this here? Is a specific modal not covered by `:set-view-id-fx`?

Reply:
this also looks like navigation issue, please log it, we can address this in follow up PR